miri

```diff
@@ -133,6 +133,20 @@ find_sysroot() {
 }
 
 ## Main
+if [[ -z "$AUTO_OPS" ]]; then
```

Comment:
I think `[[` is a bashism which we have avoided otherwise in this file.
```suggestion
if [ -z "$AUTO_OPS" ]; then
```

Reply:
I am just copy pasting stack overflow answers together XD